src/hotspot/share/gc/g1/g1RegionMarkStatsCache.hpp

```diff
@@ -98,6 +98,10 @@ class G1RegionMarkStatsCache {
 
   G1RegionMarkStatsCacheEntry* find_for_add(uint region_idx);
 public:
+  // Number of entries in the per-task stats entry. This seems enough to have a very
```

Comment:
pre-existing: please add "value" after the "This" to read "This value seems enough to...." (or "The suggested value" or something to that effect.